### ISSUE TITLE
Have a problem with commentbot

### ISSUE THREAD

**Issue by hoost1k1337** [HerrEurobeat/steam-comment-service-bot#144]
Hello 3urobeat, I'm trying to start commentbot and nothing comes out, it starts 1 account and then it just stands still, what can you advise?
![2323](https://user-images.githubusercontent.com/103311428/205931118-9b8d6783-5ed9-4957-a611-0a4a24747a75.png)

**Comment by HerrEurobeat** [comment]
Hey, thanks for opening an issue!  
I can't say for sure what's happening here but are you using proxies? I experienced the same behavior once with a proxy that had connection issues or was blocked by Steam.

**Comment by HerrEurobeat** [comment]
Since you didn't reply I am going to close this issue and hope that I answered your question.
If not, feel free to reopen this issue or please create another issue if there is something else on your mind!
